tools/deployments/__tests__/validate-pr-description.test.ts

```diff
@@ -26,28 +26,54 @@ Fixes #[insert GH or internal issue number(s)].
   - [ ] I don't know
   - [ ] Required
   - [ ] Not required because:
-- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
-  - [ ] TODO (before merge)
-  - [ ] Changeset included
-  - [ ] Changeset not necessary because:
 - Public documentation
   - [x] TODO (before merge)
   - [ ] Cloudflare docs PR(s): <!--e.g. <https://github.com/cloudflare/cloudflare-docs/pull/>...-->
   - [ ] Documentation not necessary because:
 `,
+				"[]",
 				"[]"
 			)
 		).toMatchInlineSnapshot(`
 			[
 			  "All TODO checkboxes in your PR description must be unchecked before merging",
 			  "Your PR must include tests, or provide justification for why no tests are required",
 			  "Your PR must run E2E tests, or provide justification for why running them is not required",
-			  "Your PR must include a changeset, or provide justification for why no changesets are required",
+			  "Your PR doesn't include a changeset. Either include one (following the instructions in CONTRIBUTING.md) or add the 'no-changeset-required' label to bypass this check. Most PRs should have a changeset, so only bypass this check if your change should not cause a release of any packages.",
```

Comment:
`Most PRs should have a changeset, so only bypass this check if your change should not cause a release of any packages.`

I wonder if we can be even more specific here as to what constitutes a change that requires a changeset. "should not cause a release of any packages" is still pretty wide IMO, so might be helpful for our external contributors to either narrow it down even more, or maybe give some examples

Reply:
We could maybe link specifically to https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets?

Reply:
maybe external contributors shouldn't even worry about changesets? i think we might actually save time overall by writing changesets for external contributors ourselves vs explaining our changeset conventions and chasing people up

Reply:
Added a link to the docs here to make this clearer